### DISCUSSION TITLE
Is 2.0 Coming Soon?

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#399] [General]
Sorry to rehash the topic again, but version 1.9.7 released yesterday, which means we could only have 2 more big fix updates until 2.0. Keiwan, if you have any announcements to make, we8d love to hear them.

**Comment by DinoAwakening** [comment]
I hope it doesn’t release on September or early October. Since that would lead us to get more features worthy of an actual 2.0 update ;)

**Comment by ota-works** [comment]
Let's be patient and not rush the dude. 
I don't want us to be stressing him with anticipation from our end, but I'd wish he collaborated with others to beef up Toonsquid tho.

**Comment by keiwando** [comment]
I added an FAQ entry about this question in general:
https://toonsquid.com/handbook/guides/faq/#next-update-when

As a sidenote: There is no limit to how high each of the numbers in the version number can go. The next feature update will be 1.10.0 and it will come with lots of exciting new features, but there can also still be an unlimited number of bug fix updates until then if necessary.

I have a rough idea what kind of large new feature update I might decide to call a "2.0" in the future, but I have no idea when exactly that will be.

**Reply by DWrightAnimation** [reply]
What kind of features will 1.10.0 have? Will "Out of Pegs" be one of them?

**Reply by ota-works** [reply]
Come on, man. Let's just wait and anticipate

**Reply by DinoAwakening** [reply]
Told y'all it would be 1.10 instead of a 2.0. Of course it was too early for that XD

Btw, I hope 1.10 comes with some reworks or enhancements for selection tool, or the bucket tool ;)
